standard/statements.md

```diff
@@ -604,6 +605,19 @@ Local function bodies are always reachable. The endpoint of a local function dec
 
 If the type of the argument to a local function is `dynamic`, the function to be called must be resolved at compile time, not runtime.
 
+A local function shall not be used in an expression tree.
+
+A static local function
+
+- Shall not reference instance members from an implicit or explicit `this` or `base` reference.
+- Shall not reference local variables, parameters, or non-static local functions from the enclosing scope.
+- May reference static members, type parameters, constant definitions and static local functions from the enclosing scope.
+- May use `nameof()` to reference locals, parameters, non-static local functions, `this` or `base` from the enclosing scope.
```

Comment:
```suggestion
- May reference static members, type parameters, constant definitions and static local functions from the enclosing scope.
- May use `nameof()` to reference locals, parameters, non-static local functions, `this` or `base` from the enclosing scope, but otherwise
  - shall not reference instance members from an implicit or explicit `this` or `base` reference.
  - shall not reference local variables, parameters, or non-static local functions from the enclosing scope.
```